examples/hello_world/Pipfile

```diff
@@ -4,7 +4,6 @@ verify_ssl = true
 name = "pypi"
```

Comment:
You could probably just delete this file (and Pipfile.lock) entirely if there are no dependencies.